.github/workflows/image-push-release.yml

```diff
@@ -39,6 +34,9 @@ jobs:
           flavor: |
             latest=false
 
+      - name: Discover CNI plugins version
+        run: |
+          echo "CNI_VERSION=$(echo ${{ github.ref_name }} | awk '{gsub(/-update.*$/,"")}1')" >> $GITHUB_ENV
```

Comment:
i believe we need to add a check that the exracted CNI version is as we expect.

Reply:
done in a separate step